firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift

```diff
@@ -61,6 +61,13 @@ class OnboardingMultipleChoiceButtonView: UIView, Themeable {
         imageView.isAccessibilityElement = false
     }
 
+    private lazy var stackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.alignment = .center
+        stackView.distribution = .equalSpacing
+        stackView.spacing = 7
```

Comment:
If my memory didn't fail me, I recalled that `7` was arbitrary when we discussed yesterday, is that still true? 

Side thought: maybe we can standardize spacing in our app... something to thing about once we standardize fonts...

Reply:
This was arbitrary, but I checked with Gabby and she's fine with it. However, I should move this into the constants.